examples/end-to-end-examples/sec_10k_qa/app.py

```diff
@@ -178,11 +178,10 @@ def greet(
         endpoint_url=args.llm_endpoint_url,
         inject_instruction_format=True,
         model_kwargs={
-            'max_new_tokens':
+            'output_len':
                 200,  # maximum number of response tokens to generate
-            'do_sample': False,  # perform greedy decoding
-            'use_cache': True
-            # other HuggingFace generation parameters can be set as kwargs here to experiment with different decoding parameters
+            'top_k': 1,  # perform greedy decoding
+            # other generation parameters can be set as kwargs here to experiment with different decoding parameters
```

Comment:
Any docs we can link to here? I know the ft docs are rough...

Reply:
we can link to our model handler that has description of these params.

Reply:
added a pointer to description as a comment.